spec/acceptance/suites/rpm_docker/nodesets/default.yml

```diff
@@ -44,11 +51,17 @@ HOSTS:
       - 'yum install -y epel-release'
       - 'yum install -y openssl util-linux rpm-build augeas-devel createrepo genisoimage git gnupg2 libicu-devel libxml2 libxml2-devel libxslt libxslt-devel rpmdevtools clamav clamav-update which ruby-devel rpm-devel rpm-sign'
 
+      # Openssl 1.0.2 required for gem-wrapper and ruby 2.1.0 openssl
+      - 'yum-config-manager --enable base'
```

Comment:
Just do this before we disable `base` the first time up near the `yum-utils` installation with a regular `yum install`

Reply:
I forget if I had a reason for doing it this way, but I'll try to consolidate it as you suggested.